src/app/dataset-block/metadata-block/components/event-details/components/common/dataset-kind-property/dataset-kind-property.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { BasePropertyComponent } from "../base-property/base-property.component";
+
+@Component({
+    selector: "app-dataset-kind-property",
+    templateUrl: "./dataset-kind-property.component.html",
+    styleUrls: ["./dataset-kind-property.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DatasetKindPropertyComponent extends BasePropertyComponent {
```

Comment:
It seems there are too little responsibilities for this thing to be a dedicated component.

Reply:
I deleted this component